contracts/ExampleNFT.cdc

```diff
@@ -16,13 +16,17 @@ import "MetadataViews"
 
 access(all) contract ExampleNFT: NonFungibleToken {
 
+    /// Standard Paths
+    access(all) let CollectionStoragePath: StoragePath
+    access(all) let CollectionPublicPath: PublicPath
```

Comment:
Adding new fields would get rejected during the contract update

Reply:
These aren't new. They existed in the old version, so since we're expecting people to follow this as an example for how to update their contracts, keeping them makes sense